config/env/default.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
   },
   // sessionSecret should be changed for security measures and concerns
   sessionSecret: process.env.SESSION_SECRET || 'MEAN',
-  // sessionKey is set to the generic sessionId key used by PHP applications
-  // for obsecurity reasons
+  // sessionKey is the session name and is set to the generic sessionId key
+  // used by PHP applications for obsecurity reasons
```

Comment:
The actual use of PHP cookie name for session IDs is 'PHPSESSIONID' but I thought putting that might be confusing. Alternative of 'JSESSIONID' as well so maybe we should just take out the comment and leave the generic cookie name as is.
I'll commit a change.